tests/acceptance/index.js

```diff
@@ -23,11 +23,14 @@ runTests(argv.browsers);
 async function runTests (browsers) {
   const testcafe = await createTestCafe();
   try {
-    await testcafe.createRunner()
+    const result = await testcafe.createRunner()
       .src('tests/acceptance/suites/*.js')
       .browsers(browsers)
       .startApp(`npx serve -p ${PORT} test-site/public`, 4000)
       .run({ quarantineMode: true });
+    if (result > 0) {
```

Comment:
nit: Is result the exit code of the runner? If so, I think we could exit with that code so that we don't lose that info. If not, this works!

Reply:
result is the number of failed tests here!

Reply:
ah I should rename this